### PR TITLE
hypershift: hypershift-azure update

### PR DESCRIPTION
**Pull request by LiangquanLi930**
remove 4.12 stable part, Adjust test frequency

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
@liangxia Could you help to review this PR? Thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -1465,40 +1465,6 @@ tests:
...
-    cluster_profile: azure-qe
-    env:
-      BASE_DOMAIN: qe.azure.devcluster.openshift.com
-      E2E_RUN_TAGS: '@azure-ipi and @network-ovnkubernetes and not @fips'
-      TAG_VERSION: '@4.12'
-      TEST_IMPORTANCE: Critical
-    test:
-    - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-azure-ipi-ovn-hypershift
-- as: azure-ipi-ovn-ipsec-p2-f14
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:1480]: Are those tests removal expected ?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 2]
```diff
@@ -1282,7 +1294,7 @@ tests:
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-azure-ipi-ovn
 - as: azure-ipi-ovn-hypershift-mgmt-p3-f21
-  cron: 13 17 19 * *
+  cron: 26 11 21,5 * *
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1297]: The cron should match the frequency. Update them in pair.

**Comment by LiangquanLi930** [comment]
@liangxia  Thanks for your review, Could you help to review this PR again? thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 3]
```diff
@@ -1294,18 +1306,6 @@ tests:
     - ref: cucushift-helper-service-build-info
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn-hypershift
-- as: azure-ipi-ovn-hypershift-p3-f21
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1297]: The relocation makes the tests unsorted.
```
azure-ipi-ovn-hypershift-install-p3-f21
azure-ipi-ovn-hypershift-p3-f21
```

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1297]: Sorry, let me update it. Thanks a lot!

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml [hunk 4]
```diff
@@ -10039,16 +10039,6 @@ periodics:
     job-release: "4.12"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-ipi-p3-f28-long-duration-part3-3
-  reporter_config:
```
Comment (liangxia) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml:10042]: Are those removal expected ?

Comment (LiangquanLi930) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml:10042]: That's weird, I didn't make those changes, I'll revert it

**Pull request merged**